Makefile.simde

```diff
@@ -6,6 +6,7 @@ OBJS=		kthread.o kalloc.o misc.o bseq.o sketch.o sdust.o options.o index.o chain
 PROG=		minimap2
 PROG_EXTRA=	sdust minimap2-lite
 LIBS=		-lm -lz -lpthread
+SSE4=	
```

Comment:
I found the tailing space (tab) here. It's better to remove it.

Reply:
Removed.